cmd/activity-monitor/main.go

```diff
@@ -151,14 +146,12 @@ func main() {
 			syslogU, err := url.Parse(c.GlobalString("jsonlog"))
 			if err != nil {
 				log.Fatalf("Could not parse Syslog URL: %s", err)
-				return
 			}
 
 			logger.SetEndpoint(syslogU.Scheme, syslogU.Host)
 			err = logger.Connect()
 			if err != nil {
 				log.Fatalf("Could not open remote syslog: %s", err)
-				return
```

Comment:
For development purposes you should be able to start without a remote syslog. But if the user specifies one, it should be fatal to fail to connect.